sonar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "Building docker image"
+docker build -f ./config/sonar.dockerfile -t sonar ./config/
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonar:latest
```

Comment:
And I think, wercker config will be rather different from this script... May be we leave it more or less as it is, for developer local usage. And basing on it, I will create a wercker config. In a meantime we can review existing violations.